lib/caasp.pm

```diff
@@ -71,6 +72,12 @@ sub trup_call {
         else {
             die "Confirmation dialog not shown";
         }
+        # Signature verification failed - confirmed feature by Michael Schroeder
+        if (is_caasp('kubic') && $cmd =~ /install/) {
```

Comment:
This will work, for now - but SUSE CaaSP 4.0 can be expected to have the same problem.